document procedure to kill stale ros2 daemons (#185)
osrf/rmw_zenoh: README.md

> Note: Without the zenoh router, nodes will not be able to discover each other since multicast discovery is disabled by default in the node's session config. Instead, nodes will receive discovery information about other peers via the zenoh router's gossip functionality. See more information on the session configs [below](#config).

> ### Clear stale ros2 daemons

Reviewer: ```suggestion
### Terminate ROS 2 daemon started with another RMW
```

```bash
pkill -9 -f ros && ros2 daemon stop
```

> Without this step the ROS2 CLI commands (e.g. ``ros2 node list``) may

Reviewer: ```suggestion
Without this step, ROS 2 CLI commands (e.g. `ros2 node list`) may
```

> not work properly since they could still be using a ros2 daemon that

Reviewer: ```suggestion
not work properly since they would query ROS graph information from the ROS 2 daemon that
```

> is running on a different RMW.

Reviewer: ```suggestion
may have been started with different a RMW.
```
